src/lib/types/RemoteConnectionConfig.ts

```diff
@@ -45,6 +45,8 @@ export const RemotePollerConfigSchema = z.strictObject({
 
 export type RemotePollerConfig = z.infer<typeof RemotePollerConfigSchema>;
 
-export const RemotePollerConfigMapSchema = z.record(RemotePollerConfigSchema);
+export const RemotePollerConfigMapSchema = z.record(
+  RemotePollerConfigSchema.or(z.undefined())
```

Comment:
I wonder why the `.or(z.undefined())` was added here. did you do that intentionally or was it an automated eslint change?

Reply:
I added this, not every key has a value.

Reply:
@RossWilliams every key _should_ have a value (that is the intent anyway). the `ConfigMapSchema` is the map of config id to config entry in stash:

```typescript
  "my-ftp-id": {
    // remote poller config details
  }
```

what would the use case be for not having a value to go along with this key?